frontend/src/app/render_wasm/api.cljs

```diff
@@ -423,6 +423,36 @@
   [opacity]
   (h/call internal-module "_set_shape_opacity" (or opacity 1)))
 
+(defn- translate-constraint-h
+  [type]
+  (when type
+    (case type
+      :left      0
+      :right     1
+      :leftright 2
+      :center    3
+      :scale     4)))
+
+(defn set-constraints-h
+  [constraint]
+  (when constraint
+    (h/call internal-module "_set_shape_constraint_h" (translate-constraint-h constraint))))
+
+(defn- translate-constraint-v
+  [type]
+  (when type
```

Comment:
Duplicate check: you already checking for it on
![image](https://github.com/user-attachments/assets/98a21213-6c7f-439a-9afb-fdc929834482)